### ISSUE TITLE
🐛less with --clear-screen doesn't work

### ISSUE THREAD

**Issue by kutsan** [dandavison/delta#185]
Although other `less` options seems to be working fine. less doesn't trigger repaint with `--clear-screen` after pager is closed.

```sh
export PAGER='less --clear-screen'
```

Great software by the way.

**Comment by dandavison** [comment]
Hi @kutsan, thanks for this. Correct me if I'm wrong, but is it possible that you meant `--no-init` aka `-X`, rather than `--clear-screen`? I don't know what `--clear-screen` does, but it seems that `--no-init` is the less option that controls whether or not the screen is cleared when less exits.

In any case, here are the facts:

1. If you set `BAT_PAGER=less -R` it will clear the screen when the pager exits. `PAGER` was being overridden deliberately, but `BAT_PAGER` was not.
2. However, I've just merged #181, so this overriding behavior has gone away on master, and so it will clear the screen by default on master (not released to package managers yet).
3. One does need to be a bit careful because there are several relevant settings. In particular, watch out for the `LESS` env var, as well as `BAT_PAGER` and `PAGER`: Maybe look at
    ```
    env | grep -iE '(less|bat|pager)'
    ```
4. All this behavior in delta is the same as in [bat](https://github.com/sharkdp/bat/) because delta uses bat's paging code (#181 brings delta up-to-date with bat in this respect.)

**Comment by kutsan** [comment]
Yes, you are right `-X` is probably the reason and I understand the difference. Thanks!
